### ISSUE TITLE
Need clarification on getMusiFolders request

### ISSUE THREAD

**Issue by hoffrichterm** [owncloud/music#1108]
Hello everyone,

when looking at the subsonic 1.16.1 xsd (http://www.subsonic.org/pages/inc/api/schema/subsonic-rest-api-1.16.1.xsd) the response of the GetMusicFolders endpoint is a <musicFolders> element with <musicfolder> child elements. The id attributes of the <musiFolder> child elements has to be numeric (Integer). Nextcloud Music is returning a response like this:

```
<?xml version="1.0" encoding="UTF-8"?>
<subsonic-response xmlns="http://subsonic.org/restapi" status="ok" version="1.16.1">
  <musicFolders>
    <musicFolder id="artists" name="Künstler"/>
    <musicFolder id="folders" name="Folders"/>
  </musicFolders>
</subsonic-response>
```
The endpoint URL is:
/index.php/apps/music/subsonic/rest/getMusicFolders?u=username&p=<yourpasswordhere>&v=1.16.1&c=webplayer

Maybe there are some hardcoded values or I have missed some documentation. Can you help me?

Thanks in advance
Marc

**Comment by paulijar** [comment]
There's a small deviation here between the API specification and the API implementation in the Music app. I originally didn't look carefully enough and didn't notice that the `<musicFolder>` elements have the type of `id` set as int while every other element type in the API define the `id` as string. I noticed it at some point later, but didn't care to fix it as this didn't seem to cause problems with any of the clients I have tested.
